util-test/src/commonMain/kotlin/care/data4life/sdk/util/test/Constants.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2021 D4L data4life gGmbH / All rights reserved.
+ *
+ * D4L owns all legal rights, title and interest in and to the Software Development Kit ("SDK"),
+ * including any intellectual property rights that subsist in the SDK.
+ *
+ * The SDK and its documentation may be accessed and used for viewing/review purposes only.
+ * Any usage of the SDK for other purposes, including usage for the development of
+ * applications/third-party applications shall require the conclusion of a license agreement
+ * between you and D4L.
+ *
+ * If you are interested in licensing the SDK for your own applications/third-party
+ * applications and/or if you’d like to contribute to the development of the SDK, please
+ * contact D4L by email to help@data4life.care.
+ */
+
+package care.data4life.sdk.util.test
+
+internal object Constants {
```

Comment:
could you name this as more specific, maybe CommonPath so that the constant would be named only root and we could put all other paths if needed

Reply:
Already done in #34 (see [here](https://github.com/d4l-data4life/hc-util-sdk-kmp/blob/0e0785ea342cf3f18f261bc09709b9157334f815/util-test/src/commonMain/kotlin/care/data4life/sdk/util/test/CommonResourceLoader.kt))